packages/fuselage/src/components/Card/CardControls.tsx

```diff
@@ -1,5 +1,13 @@
-const CardControls = ({ ...props }) => (
-  <div className='rcx-card__controls' {...props} />
-);
+import { useBreakpoints } from '@rocket.chat/fuselage-hooks';
+
+import Box from '../Box/Box';
+
+const CardControls = ({ ...props }) => {
+  const breakpoints = useBreakpoints();
+  const isMobile = !breakpoints.includes('sm');
+  return (
+    <Box rcx-card__controls rcx-card__controls--wrap={isMobile} {...props} />
+  );
+};
```

Comment:
```suggestion
import type { ReactNode, MouseEvent } from 'react';

type CardControlsProps = {
  children?: ReactNode;
  onClick?: (e?: MouseEvent<HTMLOrSVGElement>) => void;
};

const CardControls = ({ onClick, children }: CardControlsProps) => (
  <div className='rcx-card__controls' onClick={onClick} children={children} />
);

export default CardControls;
```

There is no need to change this component to box as you can do the conditional styling using the `on-breakpoint` mixin. Also, I've changed the `{...props}` as we should avoid this pattern (I know this is not necessarily part of you PR but it would be good if it could be added)

Reply:
Yes, this definitely looks the better way to handle this, I couldn't find the on-breakpoint mixin at that time, that's why chose to convert this to a box. And about the props, I will keep that in mind. Thanks for these suggestions.